commonknowledge/website
# Fix dependabot alerts

## Description
I've auto-updated packages with yarn upgrade --latest. For the packages that needed to be fixed manually I added to the dependencies and then specific versions to resolutions in package.json. There's 2 remaining vulnerabilities that have no patch available. I will check but maybe we could replace these packages with something else?

## Motivation and Context
Fixing security issues with out of date dependencies. 

## How Can It Be Tested?
Download branch and run yarn to install packages. Run yarn audit to see vulnerabilities. 


## Types of changes

I'm not sure if this is related to the updates ie this PR or not but I'm getting the below error when I try to run yarn develop
![Screenshot 2022-01-26 at 10 32 55](https://user-images.githubusercontent.com/43313455/151150585-95095c0e-4dfe-4ccb-accc-8949afd17847.png)
It seems like there is a wider issue with the gatsby-plugin-favicon package and it seems like you've dealt with this before with the sharp package?
If merging this branch won't break production I could dig into this as my next step on the website project?


## Checklist:
